backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/VeloxValidator.scala

```diff
@@ -21,13 +21,13 @@ import io.glutenproject.backendsapi.ValidatorApi
 
 import org.apache.spark.sql.types.{ArrayType, BinaryType, BooleanType, ByteType, DataType, DateType, DecimalType, DoubleType, FloatType, IntegerType, LongType, MapType, ShortType, StringType, StructType, TimestampType}
 import io.glutenproject.substrait.plan.PlanNode
-import io.glutenproject.validate.NativePlanValidatorInfo
+import io.glutenproject.validate.NativePlanValidationInfo
 import io.glutenproject.vectorized.NativePlanEvaluator
 
 import org.apache.spark.sql.catalyst.expressions.Expression
 import org.apache.spark.sql.execution.SparkPlan
 
-class Validator extends ValidatorApi {
+class VeloxValidator extends ValidatorApi {
 
```

Comment:
Why do we need this change? This class exists in backends-velox folder, so the scope should be apparant.

Reply:
No obvious reason for this, could revert if you insisted.

Reply:
Let's revert this change. We need to avoid duplicate prefix.